lib/engine/game/g_1880/depot.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative '../../depot'
+
+module Engine
+  module Game
+    module G1880
+      class Depot < Engine::Depot
+        def min_depot_train
+          # 2p doesn't count towards needing a train, should be ignored when checking for min
+          depot_trains.reject { |t| t.name == '2P' }.min_by(&:price)
```

Comment:
super.reject {...}.min_by(&:price)

Reply:
super returns a specific train, not an array. 
` def min_depot_train
      depot_trains.min_by(&:price)
    end`